lib/traixroute/controller/string_handler.py

```diff
@@ -284,7 +305,6 @@ def shortinlong(self, sname, lname):
 
         if sname == '' or lname == '':
             return False
-        whitelist = set('abcdefghijklmnopqrstuvwxyz1234567890 ')
```

Comment:
this line was not used

Reply:
keep it for now, I have to check if we still need string sanitization. You can add a related issue.